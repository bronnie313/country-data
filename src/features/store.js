import { configureStore } from '@reduxjs/toolkit';
import modalReducer from './modal/modalSlice';
import countryReducer from './countries/countriesSlice';

const store = configureStore({
  reducer: {
    modal: modalReducer,
    country: countryReducer,
  },
});

export default store;
