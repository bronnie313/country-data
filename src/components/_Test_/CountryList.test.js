import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../../features/store';
import CountryList from '../countryList';

describe('home component', () => {
  it('testing if page matches', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <CountryList />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
