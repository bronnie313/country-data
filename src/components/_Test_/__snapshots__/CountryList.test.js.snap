// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home component testing if page matches 1`] = `
<section>
  <div
    className="countries"
  />
</section>
`;
